<?xml version="1.0" encoding="UTF-8"?>

<!-- 
 * Copyright (C) 2013 ~ 2021 KylinSoft Ltd.
 *
 * Authors:
 *  Kobe Lee    lixiang@kylinos.cn
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; version 3.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->

<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Launch window of kmre</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 145, 212);</string>
  </property>
  <widget class="QLabel" name="text_label">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>14</y>
     <width>342</width>
     <height>92</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: rgb(255, 255, 255);
        background-color: rgba(0,0,0,0);
     }</string>
   </property>
   <property name="text">
    <string>Preparing Kmre Env</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
        border: 0px solid grey;
        border-radius: 0px;
        text-align: center;
        background-color: rgba(0,0,0,0);
     }

     QProgressBar::chunk {
        background-color: rgba(0,0,0,25%);
        width: 20px;
        /*margin:0.5px*/
     }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>29</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     background-color: rgba(0,0,0,0);
 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>logo_label</zorder>
  <zorder>text_label</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
